data/overview.yaml

```
@@ -59,106 +104,135 @@ Groups:
     unitary group: ''
     special unitary group: ''
   representation theory for finite groups:
-    representations of abelian groups: ''[
+    representations of abelian groups: ''
+    dual groups: ''
     Maschke theorem: ''
     orthogonality of irreducible characters: ''
     Fourier transform for finite abelian groups: ''
     convolution: ''
+    class function over a group: ''
+    characters of a finite dimensional representation: ''
+    orthonormal basis of irreducible characters: ''
     examples for groups with a small cardinal: ''
 
 3. 
-Rings: 
-  Unitary rings:
+Rings:
+  ring:
+  Unitary ring:
   ring morphisms: 
   subrings: 
-  The ring Zn on the integers: 
+  The ring Z on the integers: 
+  Product of rings:
   Ideal of a commutative ring: 
   quotient rings: 
-  first ideals: 
+  prime ideals: 
   maximal ideals: 
   Chinese remainder theorem: 
-  Commutative ring algebra:
+  Algebra:
+    algebra over a commutative ring:
+    associative algebra over a commutative ring:
   Divisibility in commutative integral rings: 
   Irreducible elements: 
   invertible elements: 
-  first elements among them: 
-  Factorial rings: 
+  Coprimes elements: 
+  Unique factorization domain (UFD): 
   Greatest common divisor: 
-  least common multiple:
-  Factorability of A[X] when A is a factorial ring:
+  Least common multiple:
+  A[X] is a UFD when A is a UFD:
   Principal ring: 
```

```suggestion
  Principal ideal ring: 
```
Or maybe principal ideal domain?

---

I think, it should be principal ideal domain

---

I don't see any reason to ignore principal rings that have zero divisors.

---

Let's consider this resolved.